tests/wpt/web-platform-tests/html/semantics/forms/textfieldselection/defaultSelection.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<meta charset="utf-8">
+<title></title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<textarea>g
+</textarea>
+<script>
+test(function() {
+    let textarea = document.querySelector('textarea');
+    assert_equals(textarea.selectionStart, 0);
+    assert_equals(textarea.selectionEnd, 0);
+},"Default selectionStart and selectionEnd");
```

Comment:
Let's add another test for `<input type="text" value="foo">` as well.

Reply:
Okay, i'll do that.